electron/shortcuts/nonsteamgame.ts

```diff
@@ -207,6 +196,7 @@ async function addNonSteamGame(props: {
       error,
       adding: true
     })
+    return
   } else {
```

Comment:
Well now you have to actually get rid of the `else` to get rid of the indentation